src/openrct2/GameState.h

```diff
@@ -36,6 +36,7 @@ namespace OpenRCT2
         uint32_t ParkSize;
         money64 ParkValue;
         money64 ParkValueHistory[FINANCE_GRAPH_SIZE];
+        uint8_t ParkRatingHistory[32];
```

Comment:
I’d like to see a constant for this. If you don’t want to do that in this PR, I’d like you to create an issue instead.

Reply:
Was thinking that too, but haven't found any. All places that loop over this array simply use its length, no constant. Unless I overlooked a constant, one has to be introduced most likely. Because I have very little time the next couple days, I'd like to leave that for another PR.

Reply:
Was thinking that too, but haven't found any. All places that loop over this array simply use its length, no constant. Unless I overlooked a constant, one has to be introduced most likely. Because I have very little time the next couple days, I'd like to leave that for another PR.

Reply:
https://github.com/OpenRCT2/OpenRCT2/issues/21346